ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Config.scala

```diff
@@ -30,6 +31,7 @@ object Config {
     verbose = false,
     commandSubmissionTtlScaleFactor = 1.0,
     timeoutScaleFactor = 1.0,
+    concurrentTestRuns = None,
```

Comment:
You could also set the default here directly, then we don't need the `Option` in the type and `getOrElse` later.

Reply:
Sounds good to me. I wasn't sure if this was OK as everything else in there is a real constant, but if you're happy, I'm happy. 😄